workflow-controller/values.yaml

```diff
@@ -0,0 +1,55 @@
+global:
+  clusterScope:
+    enabled: false
+  autoUpgrade:
+    enabled: true
+  infraId: ""
+  accessKey: ""
+  workflowInstanceId: ""
+  accountId: ""
+  serverAddress:  https://app.harness.io/gratis/chaos/kserver/api
+
```

Comment:
Some values can be removed from here - 
- autoUpgrade
- infraId
- accessKey
- accountId
- serverAddress

Reply:
They are global overrides, they are good to have in sub-chart values.yaml as well, as some of them are required key in the sub-charts